### PR TITLE
Add more html tags in html-cheatsheet.md file

### PR DESCRIPTION
**Pull request by Manoj0Marmat**
Signed-off-by: Manoj Marmat <manojrajput762000@gmail.com>

### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by Manoj0Marmat** [comment]
@crescentpartha hey please review it

File: CheatSheets/html-cheatsheet.md [hunk 1]
```diff
@@ -39,19 +40,21 @@
...
 | `<dfn> … </dfn>`               | Tag dedicated for definitions.                                                                                                 |
 | `<code> … </code>`             | This is used to display code snippets within a paragraph.                                                                      |
+
+| `<aside> … </aside>`           | Its a Block level element that defines content aside from the content in which it is Placed.                                   |
+
 | `<sub> … </sub>`               | Used for writing a subscript (smaller font just below the mid-point of normal font).                                           |
 | `<sup> … </sup>`               | Similar to the above tag, but for superscripting.                                                                              |
 | `<small> … </small>`           | Reduces text size. In HTML5, it often refers to redundant or invalid information.                                              | 
 | `<strong> … </strong>`         | This element indicates that its contents have strong importance or urgency. Browsers usualy render the contents in bold type.  |
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:55]: Those tags break the table format. Please Fix it.

File: CheatSheets/html-cheatsheet.md [hunk 2]
```diff
@@ -66,14 +69,30 @@
...
 
+## Formatting
+
+| Command                  | Description                                          |
+| ------------------------ | ---------------------------------------------------- |
+| `<b> ... </b>`           | Defines bold text                                    |
+| `<em> ... </em>`         | Defines emphasized text                              |
+| `<i> ... </i>`           | Defines a part of text in an alternate voice or mood |
+| `<small> ... </small>`   | Defines smaller text                                 |
+| `<strong> ... </strong>` | Defines important text                               |
```
Comment (crescentpartha) [inline CheatSheets/html-cheatsheet.md:87]:
```suggestion
```
Those tags already exist. Please `delete` the `older ones`.

**Review by crescentpartha — CHANGES_REQUESTED** [review]
First of all, need to fix the broken code, then modify the required changes.

File: CheatSheets/html-cheatsheet.md (hunk 1 above)
Comment (Manoj0Marmat) [reply CheatSheets/html-cheatsheet.md:55]: okay

File: CheatSheets/html-cheatsheet.md (hunk 2 above)
Comment (Manoj0Marmat) [reply CheatSheets/html-cheatsheet.md:87]: sure got it

**Pull request closed without merging**